pkg/syncer/metadatasyncer.go

```diff
@@ -91,6 +91,8 @@ var (
 	isMultiVCenterFssEnabled bool
 	// nodeMgr stores the manager to interact with nodeVMs.
 	nodeMgr node.Manager
+	// isPodVMOnStretchSupervisorFSSEnabled is true when PodVMOnStretchedSupervisor FSS is enabled.
+	isPodVMOnStretchSupervisorFSSEnabled bool
```

Comment:
where do you set value of `isPodVMOnStretchSupervisorFSSEnabled`?

Reply:
My bad, added it in `InitMetadataSyncer` function